Os/CMakeLists.txt

```diff
@@ -97,26 +98,33 @@ endif()
 register_fprime_module()
 
 require_fprime_implementation(Os/File)
+require_fprime_implementation(Os/FileSystem)
+require_fprime_implementation(Os/Directory)
+require_fprime_implementation(Os/Mutex)
 # require_fprime_implementation(Os/Task) # should be added in
-require_fprime_implementation(Os/Mutex) 
+# require_fprime_implementation(Os/Console) # should be added in
 
 
 ### UTS ### Note: 3 separate UTs registered here.
 set(UT_SOURCE_FILES
   "${CMAKE_CURRENT_LIST_DIR}/test/ut/OsQueueTest.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/test/ut/OsTestMain.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/test/ut/IntervalTimerTest.cpp"
-  "${CMAKE_CURRENT_LIST_DIR}/test/ut/OsFileSystemTest.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/test/ut/OsValidateFileTest.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/test/ut/OsSystemResourcesTest.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/test/ut/OsMutexBasicLockableTest.cpp"
 )
 register_fprime_ut()
 if (BUILD_TESTING)
-    foreach (TEST IN ITEMS StubFileTest PosixFileTest StubMutexTest PosixMutexTest) # add? : StubTaskTest PosixTaskTest
-       if (TARGET "${TEST}")
-           add_dependencies("Os_ut_exe" "${TEST}")
-       endif()
+    foreach (TEST IN ITEMS 
```

Comment:
We need to take an action to clean this up.  Outside this PR.

Reply:
https://github.com/nasa/fprime/issues/2904